src/veh_interact.cpp

```diff
@@ -814,6 +834,29 @@ void veh_interact::do_install()
     display_name();
 }
 
+void veh_interact::redraw() const
+{
+    redrawwin(w_grid);
+    redrawwin(w_mode);
+    redrawwin(w_msg);
+    redrawwin(w_disp);
+    redrawwin(w_parts);
+    redrawwin(w_stats);
+    redrawwin(w_list);
+    redrawwin(w_details);
+    redrawwin(w_name);
```

Comment:
clearok/wrefresh pair doesn't work in console and probably tiles(I tried to call clearok from redrawwin - it didn't work). I didn't find other routines in `cursesport`  that just change touch flag and added redrawwin which works in console and tiles.